### PR TITLE
Change the way how to pay about >1024 transaction. (2x)

### PR DESCRIPTION
**Pull request by lightszero**
#neo-project/neo-cli#303

pay for every extrablock instread  every bytes

in default extrablock=1024 bytes.
feeperextrablock=0.0124.gas

example:
txsize  < 1024, pay 0 block,free
txsize =1025  ~  2048 bytes, pay 1 block    0.01024gas
txsize =2049  ~  3072 bytes, pay 2 block    0.02048gas

if you want pay by every extrabytes
set extrablock=1 bytes
feeperextrablock =0.0001gas

### CODE REVIEW

**Comment by igormcoelho** [comment]
@lightszero could we accept these tests here first? https://github.com/neo-project/neo-plugins/pull/101

Then we update the tests on this branch to see how the new prices fit.

**Comment by vncoelho** [comment]
Accepted and merged, those tests are great, @igormcoelho.

**Comment by vncoelho** [comment]
@lightszero, what are the advantages of this? Are you afraid of any attack?
Are you planning this for both 2x and 3.0+?

**Comment by igormcoelho** [comment]
@lightszero I added your proposed option as "standard" (VerifySizeLimits2) and kept old one too (VerifySizeLimits1). Now we can discuss and create more tests for your proposal.

**Comment by igormcoelho** [comment]
@shargon could you fix the indent here for us? poor editor here hahaha

**Comment by jasonliu91** [comment]
This policy is for 2x ,use kb as fee unit can avoid the utxo loop calculation problem of the previous byte unit policy.

**Comment by igormcoelho** [comment]
I think block calculation is a good idea. But since old formula has been around for a while , we should propose another obe which is compatible.. with lower or same prices.

**Comment by igormcoelho** [comment]
@lightszero what about charging at every 512 bytes, and use FLOOR instead of CEILING? This reduces the UTXO combinations and effectively keep prices nearly the same or less (at small intervals).
`fee = floor((txsize - 1024)/512)*pricePer512Chunk + basePrice`

**Comment by lightszero** [comment]
> @lightszero, what are the advantages of this? Are you afraid of any attack?
> Are you planning this for both 2x and 3.0+?

because they donot like #73 

so this is another try

File: SimplePolicy/SimplePolicy/config.json [hunk 1]
```diff
@@ -2,8 +2,10 @@
   "PluginConfiguration": {
     "MaxTransactionsPerBlock": 500,
     "MaxFreeTransactionsPerBlock": 20,
-    "MaxFreeTransactionSize": 1024,
     "FeePerExtraByte": 0.00001,
```
Comment (vncoelho) [inline SimplePolicy/SimplePolicy/config.json]: It is a little bit strange to still keep `FeePerExtraByte` here.

However, the old `VerifySizeLimits1` limit was kept.

**Comment by shargon** [comment]
Is for neo2, i think that we shouldn't change the economic model in neo 2

**Comment by vncoelho** [comment]
I also think so, @shargon, maybe let's close this one and think about adjusting it for NEO3.
What do you think @lightszero and @erikzhang ?

**Comment by lock9** [comment]
Hello @neo-project/core, what should we do with this?

**Comment by vncoelho** [comment]
@lightszero, I think we can close this, right?
There is no more need of changes on NEO2x.
In my opinion, it was a good change but it was not merged in time.

However, fell free to reopen.

**Comment by vncoelho** [comment]
Reopened due to https://github.com/neo-project/neo/pull/1763, if we are going to charge for tx size better to charge this way.

**Pull request closed without merging**